src/frontend/src/handler/create_mv.rs

```diff
@@ -114,18 +114,26 @@ pub async fn handle_create_mv(
 ) -> Result<RwPgResponse> {
     let session = context.session_ctx.clone();
 
+    let is_independent_compaction_group;
     let (table, graph) = {
         {
             let catalog_reader = session.env().catalog_reader().read_guard();
             let (schema_name, table_name) = Binder::resolve_table_name(name.clone())?;
+            is_independent_compaction_group = session.config().get_create_compaction_group_for_mv();
             catalog_reader.check_relation_name_duplicated(
                 session.database(),
                 &schema_name,
                 &table_name,
             )?;
         }
 
-        let (plan, table) = gen_create_mv_plan(&session, context.into(), query, name, false)?;
+        let (plan, table) = gen_create_mv_plan(
+            &session,
+            context.into(),
+            query,
+            name,
+            is_independent_compaction_group,
```

Comment:
function `gen_create_mv_plan` can get the `session`, can we remove the variable `is_independent_compaction_group`, and inline this logic

Reply:
> function `gen_create_mv_plan` can get the `session`, can we remove the variable `is_independent_compaction_group`, and inline this logic

thank you. it's a mistake.